src/base/Checker.ml

```diff
@@ -160,8 +160,8 @@ let check_patterns_lmodule e rlibs elibs =
     @@ sprintf "\n[Pattern Check]:\n library module is successfully checked.\n";
   res
 
-let check_sanity m rlibs elibs =
-  let res = SC.contr_sanity m rlibs elibs in
+let check_sanity m cg rlibs elibs =
```

Comment:
Let's call this `call_graph` or something like this, it will be easier to understand what goes on here later on.